apps/storybook/package.json

```diff
@@ -0,0 +1,51 @@
+{
+  "name": "storybook",
```

Comment:
I had a flood of these warning and it took ages to load up storybook...

Reply:
Also, built docker, run `yarn test`, but all tests failed.
Am I missing something?

Reply:
it needs storybook `build` first. where did you run `yarn test`? i think build should automatically happen if you run it from root using `yarn test --filter=storybook`

Reply:
Running `yarn test --filter=storybook` from root is all that's needed after I improved the turbo pipeline in [`5aa3432`](https://github.com/iTwin/iTwinUI-react/pull/660/commits/5aa343282eeb6f7eb44a3feaf883c4318345c1a8).

For running from `apps/storybook`, you still need to manually build storybook first. If you want I could add an alias here, something like:
```
"yarn test:full": "yarn build && yarn test"
```

Reply:
We must document all of that..